model-mesh/runtimes/kustomization.yaml

```diff
@@ -27,7 +27,7 @@ images:
 
   - name: ovms-1
     newName: openvino/model_server
-    newTag: "2022.1"
+    newTag: "2022.3"
```

Comment:
Does this change actually change what is created by odh-dashboard? @lucferbux fyi^^